app/pages/lab/tutorial.cjsx

```diff
@@ -12,6 +12,7 @@ module.exports = React.createClass
         <p>This is the place to give the volunteers a preview of the data they'll be working with and of the steps they'll be taking to make classifications. It's also a good place to mention any common "gotchas" users might face.</p>
         <p>However, it's also important to keep this as short as possible so volunteers can get started classifying as soon as possible!</p>
         <p><strong>Tutorials should be linked to a workflow on its workflow editor page to pop up for your project.</strong></p>
+	<p> <strong> You can reorder the stepss in your tutorial be clicking and dragging on the left gray tab </strong> </p>
```

Comment:
Looks like there are a couple of typos here: `steps` instead of `stepss` and `by clicking` instead of `be clicking`